.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,79 @@
+{
+	"name": "Home Ops",
+	"image": "mcr.microsoft.com/devcontainers/base:debian",
+	"containerEnv": {
+		"KUBECONFIG": "${containerWorkspaceFolder}/kubeconfig",
+		"SOPS_AGE_KEY_FILE": "${containerWorkspaceFolder}/.age.key"
+	},
+	"postCreateCommand": {
+		"deps": "task deps"
+	},
+	"features": {
+		// Upgrade OS packages
+		"ghcr.io/devcontainers/features/common-utils:2": {
+			"version": "latest",
+			"installZsh": false,
+			"installOhMyZsh": false
+		},
+		"ghcr.io/devcontainers-contrib/features/go-task:1": {
+			"version": "3.31.0"
+		},
+    "ghcr.io/devcontainers-contrib/features/ansible:2": {
+			"version": "2.16.0"
+		},
```

Comment:
Does this install ansible? We shouldn't need to do that since `task deps` will pull that in and set it up

Reply:
It installs python and anisble. Could change it to just a direct python install then let task deps do the rest of the python work.

Reply:
I would go that route if possible.

Reply:
Looks like the python devcontainers feature "helpfully" installs more vscode extensions (see https://github.com/devcontainers/features/blob/d53b9d1816bac7a62f9ba5f20723eef428042d1f/src/python/devcontainer-feature.json#L64) and seems like stopping that from happening is still an open feature request with devcontainers (see https://github.com/devcontainers/features/issues/386).

Personally I wouldn't want those surprise extensions.